hazelcast/src/main/java/com/hazelcast/map/impl/querycache/subscriber/operation/DestroyQueryCacheOperation.java

```diff
@@ -29,14 +29,16 @@
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.spi.EventService;
+import com.hazelcast.spi.impl.operationservice.MutatingOperation;
 import com.hazelcast.util.ExceptionUtil;
 
 import java.io.IOException;
 
 /**
  * This operation removes all {@code QueryCache} resources on a node.
  */
-public class DestroyQueryCacheOperation extends MapOperation {
+public class DestroyQueryCacheOperation
```

Comment:
Same as others, not sure if this is internal or direct user invocation. Also, not to repeat myself, same in `MadePublishableOperation`, `PublisherCreateOperation`, `ReadAndResetAccumulatorOperation` and `SetReadCursorOperation`

Reply:
fixed